scss/utilities/_json-api.scss

```diff
@@ -66,44 +71,40 @@
 /// ### Colors
 ///
 /// Each color has an access name and value.
-/// For us, that is often a map-key string,
-/// and Sass hex/rgba output color:
+/// The key will be used to identify the correct data
+/// for a given color-palette,
+/// and the data includes color name:value pairs,
+/// both in string format.
 ///
-/// ```scss
```

Comment:
Why remove the `scss` language here? Without it, our syntax highlighter interprets this as `bash`.